release/airflow/cloud_tag.template.json

```diff
@@ -9,10 +9,22 @@
       "args": [ "chmod", "u+x", "./istio_checkout_code.sh" ]
     },
     {
+      "volumes": [
+	{
+          "name": "buildoutput",
+          "path": "/output"
+	}
+      ],
```

Comment:
where did output go before?

Reply:
it was copied only when /output is present. some steps did not need it